cancermuts/datasources.py

```diff
@@ -1404,8 +1427,106 @@ def _get_gnomad_data(self, gene_id, reference_genome, dataset):
         variants['genome_af'] = variants['genome_ac'] / variants['genome_an']
         variants['total_af'] = variants['total_ac'] / variants['total_an']
 
+        variants[['popmax_exome_ac',  'popmax_exome_an',  'popmax_exome_af',
+              'popmax_genome_ac', 'popmax_genome_an', 'popmax_genome_af',
+              'popmax_tot_ac',    'popmax_tot_an',    'popmax_tot_af']] = variants.apply(self.get_popmax_af, axis=1)
+
         return variants
 
+    def get_popmax_af(self, variants, pops=['afr', 'eas', 'nfe', 'amr', 'sas']):
```

Comment:
can we
* change the name of the method according to the "private method" naming convention (so it should start with `_`)
* At the beginning of the file, under the copyright section, add where you took this from - this is necessary because the code is licensed under GPL
* add yourself at the beginning in the copyright section, with a line similar to mine (`(c) 2018 Matteo Tiberti <matteo.tiberti@gmail.com>`). We should remember to do this in every file you modify so you're properly credited :) I forgot to point it out in the previous PRs, apologies! We can add it retroactively